sanic_openapi/swagger.py

```diff
@@ -14,14 +14,6 @@
 dir_path = os.path.dirname(os.path.realpath(__file__))
 dir_path = os.path.abspath(dir_path + '/ui')
 
-
-# Redirect "/swagger" to "/swagger/"
-@blueprint.route('', strict_slashes=True)
-def index(request):
-    return redirect("{}/".format(blueprint.url_prefix))
-
-
-blueprint.static('/', dir_path + '/index.html', strict_slashes=True)
```

Comment:
In readme:
> You'll now have a Swagger UI at the URL /swagger/ and an OpenAPI 2.0 spec at /swagger/swagger.json. Your routes will be automatically categorized by their blueprints.

Please revert the part of `blueprint.static('/', dir_path + '/index.html', strict_slashes=True)` or the endpoint `/swagger/` will not work anymore.

Reply:
can open new request again, i will clone the latest open api